hawkbit-core/src/main/java/org/eclipse/hawkbit/artifact/repository/AbstractArtifactRepository.java

```diff
@@ -55,16 +55,22 @@ public AbstractDbArtifact store(final String tenant, final InputStream content,
         }
 
         String tempFile = null;
-        try (final DigestInputStream inputstream = wrapInDigestInputStream(content, mdSHA1, mdMD5, mdSHA256)) {
+        try (final DigestInputStream inputStream = wrapInDigestInputStream(content, mdSHA1, mdMD5, mdSHA256)) {
 
-            tempFile = storeTempFile(inputstream);
+            tempFile = storeTempFile(inputStream);
```

Comment:
this can potentially throw Quota exceeded exception for the artifact that is already present